test_app.py

```diff
@@ -28,6 +30,41 @@ def test_game_instructions(self):
         output = captured_output.getvalue().strip()
         self.assertEqual(output, validate_instructions)
 
+    # Game Board
+
+    # Draw Board
+    def test_initialize_board(self):
+        expected_board = [["1", "2", "3"], ["4", "5", "6"], ["7", "8", "9"]]
```

Comment:
This is good. If we wanted to head more into an OOP style, can we encapsulate the `Board` class further so that we communicate through sending messages rather than shared knowledge data structures? We could say that our `Board` could be encapsulated further, or that some of the internals of our class are leaking a little bit.

Reply:
So the TicTacToe class that emerged from the board class has been trimmed down and now only controls the gameplay loop (and creating players - but that may be changed...). TicTacToe now initiates an instance of the Board class that controls the functions of creating and modifying the board. Including move validation and win condition checks, but this seemed the most logical place to put them.